ansible/Ansible.common.mk

```diff
@@ -31,6 +27,14 @@ ansible/dependencies: $(ROLES_PATH) $(COLLECTIONS_PATH)
 	ansible-galaxy role install -r $(REQUIREMENTS_FILE) -p $(ROLES_PATH)
```

Comment:
Are we using the new Fargate CI runner image? If so, we can now remove the installation of the SSM plugin 🎉

Reply:
Good point